go/apps/http_api_nostream/tests/test_resource.py

```diff
@@ -81,39 +84,88 @@ def test_many_errors(self):
         self.assert_no_attribute(opts, "odd")
 
 
+class TestMsgCheckHelpers(TestCase):
+    def test_is_unicode(self):
+        self.assertTrue(MsgCheckHelpers.is_unicode(u"123"))
+        self.assertFalse(MsgCheckHelpers.is_unicode(None))
+        self.assertFalse(MsgCheckHelpers.is_unicode("123"))
```

Comment:
I see all the existing tests use `assertTrue()` and `assertFalse()`. If we do make the switch, we should probably just update the methods we're touching anyway.